src/main/java/com/synopsys/integration/detect/tool/detector/inspectors/nuget/NugetRuntimeResolver.java

```diff
@@ -0,0 +1,87 @@
+package com.synopsys.integration.detect.tool.detector.inspectors.nuget;
+
+import java.io.File;
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.lang3.math.NumberUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.synopsys.integration.detectable.detectable.exception.DetectableException;
+import com.synopsys.integration.detectable.detectable.executable.ExecutableOutput;
+import com.synopsys.integration.detectable.detectable.executable.ExecutableRunner;
+import com.synopsys.integration.detectable.detectable.executable.ExecutableRunnerException;
+
+public class NugetRuntimeResolver {
```

Comment:
I'd rather we separated the invocation of the command from the parsing and matching - it would make the unit tests simpler.